src/dlr_relayvm.cc

```diff
@@ -48,10 +59,44 @@ void RelayVMModel::SetupVMModule() {
                     "to override TVM allocations. Using default allocators.";
   }
 
-  tvm::runtime::Module lib = tvm::runtime::Module::LoadFromFile(path_->model_lib);
-  std::ifstream relay_ob(path_->relay_executable, std::ios::binary);
-  std::string code_data((std::istreambuf_iterator<char>(relay_ob)),
-                        std::istreambuf_iterator<char>());
+  std::string code_data;
+  std::string model_lib_path;
+  std::string metadata_path;
+  for (DLRModelElem el: model_elems) {
+    if (el.type == DLRModelElemType::RELAY_EXEC) {
+      if (el.path != nullptr) {
+        std::ifstream relay_ob(el.path, std::ios::binary);
+        std::string code_data_((std::istreambuf_iterator<char>(relay_ob)),
+                               std::istreambuf_iterator<char>());
+        code_data = code_data_;
```

Comment:
We can use `code_data.assign((std::istreambuf_iterator<char>(relay_ob)),
                               std::istreambuf_iterator<char>());`

assign is basically like calling constructor

Reply:
fixed